content/testing-yaml/testing.md

```diff
@@ -21,37 +23,66 @@ npm test
 For non-UI tests or unit testing:
 
 ```bash
-./gradlew test
+- name: Test
+  script: ./gradlew test
+  test_report: app/build/test-results/**/*.xml
 ```
 
 UI tests (also known as instrumented tests):
 
 ```bash
-./gradlew connectedAndroidTest
+- name: Launch emulator
+  script: |
+    cd $ANDROID_HOME/tools
+    emulator -avd emulator &
+    adb wait-for-device
+- name: Test
+  script: |
+    set -e
+    ./gradlew connectedAndroidTest
+    adb logcat -d > emulator.log
 ```
 
-## Native iOS
+**Tip:**: you can save the emulator lor with `adb logcat -d > emulator.log` command.
```

Comment:
```suggestion
**Tip:**: you can save the emulator log with `adb logcat -d > emulator.log` command.
```

Reply:
fixed in 2e33bafc3592fa7408d75ac9459306eb97d56a8d